woocommerce/wc-admin
# Activity Panel: set reviews unread indicator based on real data

Fixes #1819.

Uses real data in order to show or hide the unread indicator next to the _Reviews_ Activity Panel tab.

### Screenshots
![reviews3](https://user-images.githubusercontent.com/3616980/54705264-af23f100-4b3c-11e9-8b9f-d354c00dce11.gif)

### Detailed test instructions:
- Create a product review.
- Go to any `wc-admin` page and verify an unread indicator (red dot) appears next to the _Reviews_ tab of the Activity Panel.
- Click on the tab and verify the new review is also marked with a red dot.
- Close the panel and verify the red dot disappeared from the tab. Click on it again and verify the review doesn't have the red dot anymore either.
